### DISCUSSION TITLE
Canale YouTube e tutorial su come installare Giua@School su AlterVista.

### DISCUSSION THREAD

**Discussion by registro-giuaschool** [iisgiua/giuaschool#740] [Idee e proposte]
Si il canale YouTube dove è stato caricato il tutorial per installare Giua@School su AlteVista. 
Vi saluto e spero vi sia stato d'aiuto. Per ogni informazione potete aprire una segnalazione di errore, discussione oppure inviare una mail a registroelettronicogiuaschool@gmail.com

Link YouTube: https://www.youtube.com/@RegistroElettronicoGiuaSchool

**Comment by trinko** [comment]
Ottimo!
Grazie.

**Reply by registro-giuaschool** [reply]
@trinko Se sarà possibile, e se sei d'accordo, potresti aggiungermi nella sezione dei contributori del progetto Giua@School?

**Comment by trinko** [comment]
La sezione _Contributors_ è gestita automaticamente dalla piattaforma GitHub ed è basata sui contributi allo sviluppo del codice (_pull requests_).

**Reply by registro-giuaschool** [reply]
Per essere aggiunti, si fa richiesta su pull request?

**Reply by registro-giuaschool** [reply]
Poi ti chiedo @trinko di rispondermi sull'altra discussione per il tutorial su come trovare l'errore nei file log.

**Reply by trinko** [reply]
> Per essere aggiunti, si fa richiesta su pull request?

No, non mi sono spiegato.
Github gestisce i progetti software open source: chiunque può proporre una modifica all'applicazione, scrivendo il codice relativo. La _pull request_ prende le modifiche fatte sul codice e le rende visibili al _maintener_ del progetto (in questo caso io). A questo punto il _mantainer_ fa le necessarie verifiche e se approva la modifica la fa entrare nel codice ufficiale del progetto. Se questo avviene, GitHub tiene traccia di chi ha proposto la modifica che viene citato come _contributor_.

Se vuoi sapere di più sulle pull request di GitHub puoi vedere ad esempio questo tutoria:
[Github: guida per principianti per contribuire ad un progetto](https://medium.com/acadevmy/github-guida-per-principianti-per-contribuire-ad-un-progetto-757f9e619149)

**Comment by trinko** [comment]
Comunque, questo post l'ho messo in evidenza nelle Discussioni, in modo che il link al canale Youtube sia ben visibile.
